src/components/SubNavbar.js

```diff
@@ -0,0 +1,227 @@
+import React from "react";
+import AppBar from "@material-ui/core/AppBar";
+
+//Icons
+import DashboardIcon from "@material-ui/icons/Dashboard";
+import KeyboardArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
+import ViewCarouselIcon from "@material-ui/icons/ViewCarousel";
+import WidgetsIcon from "@material-ui/icons/Widgets";
+import BuildIcon from "@material-ui/icons/Build";
+import ExtensionIcon from "@material-ui/icons/Extension";
+import AssignmentTurnedInIcon from "@material-ui/icons/AssignmentTurnedIn";
+import ShoppingCartIcon from "@material-ui/icons/ShoppingCart";
+import BallotIcon from "@material-ui/icons/Ballot";
+import EmailIcon from "@material-ui/icons/Email";
+import ForumIcon from "@material-ui/icons/Forum";
+import ListAltIcon from "@material-ui/icons/ListAlt";
+import AssessmentIcon from "@material-ui/icons/Assessment";
+import AppsIcon from "@material-ui/icons/Apps";
+import MapIcon from "@material-ui/icons/Map";
+import TodayIcon from "@material-ui/icons/Today";
+import PeopleAltIcon from "@material-ui/icons/PeopleAlt";
+import CropIcon from "@material-ui/icons/Crop";
+import VideoLibraryIcon from "@material-ui/icons/VideoLibrary";
+import AddBoxIcon from "@material-ui/icons/AddBox";
+
+export default function SubNavbar() {
+  return (
+    <div>
+      <AppBar position="static" className="SubNavbar">
+        <ul className="Menu">
+          <li className="MenuItem">
+            <a href="/" className="Dropdown">
+              <DashboardIcon color="action" />
+              <span className="MenuTitle">General</span>
+              <KeyboardArrowDownIcon
+                color="action"
+                style={{ fontSize: "16px" }}
+              />
+            </a>
+            <ul className="DropdownMenu">
```

Comment:
- extract menu & structure in separate data structure and generate the template dynamically
{
   name: 'Dashboard',
   icon: {
      type: 'ListAltIcon', => <ListAltIcon data={item.children}
      size: 'small'
   },
  options: [
     {
          name: "Page 1",
          icon: 'AssesmentIcon'
      }
  ]
}